### ISSUE TITLE
Subprocess return exit code -9

### ISSUE THREAD

**Issue by kforti** [dask/dask-jobqueue#264]
I am attempting to run a subprocess in parallel using dask. However, when I submit the subprocess to a cluster it retruns a CalledProcessError and a non-zero exit status -9.

I tried running the same subprocess in an interactive job with the same memory allocated that each of my workers has and the subprocess runs just fine. Why is this subprocess not working properly when I try to run it through one of my worker nodes?

This is the subprocess function:
```
        def func(command):
            process = subprocess.check_output(command)
            return
```
Each worker has 1 core and 5 GB of memory (The subprocess is expected to use less than 2 GB of memory)

**Comment by guillaumeeb** [comment]
Any advance here @kforti? Did you try @lesteve suggestions?

**Comment by kforti** [comment]
@guillaumeeb yes, I have made progress on this. Thank you for the suggestions @lesteve. I tried your suggestions and a few other things. What I noticed was that when I manually submitted the job used by dask_jobqueue to start workers, my lsf cluster reported that no memory was selected, so it defaulted to 1 GB per job (w/ 4 workers). The process was being terminated due to a memory shortage.
The solution, I had to add an extra job parameter for my lsf cluster, which assigns memory through #BSUB -R rusage[mem=2048], and ignores #BSUB -M
